extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -268,16 +267,18 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
         DicomMetadataStore.studyLoaded(study);
       }
 
-      // Google Cloud Healthcare doesn't return StudyInstanceUID, so we need to add
-      // it manually here
-      seriesSummaryMetadata.forEach(aSeries => { aSeries.StudyInstanceUID = StudyInstanceUID })
-
       DicomMetadataStore.addSeriesMetadata(seriesSummaryMetadata, madeInClient);
 
-      const numberOfSeries = seriesPromises.length;
-      seriesPromises.forEach(async (seriesPromise, index) => {
-        const instances = await seriesPromise;
-        storeInstances(instances);
+      const numberOfSeries = seriesSummaryMetadata.length;
+      seriesSummaryMetadata.forEach(async (series, index) => {
+        const instancesResponse = await nlApi.get("http://localhost:8000/api/instances/?series_id=" + series.id);
+        const instances = instancesResponse.data.results.map(instance => ({...instance, ...series, ...patientResponse.data.results[0], ...studyResponse.data.results[0], ...instance.overlay_data }))
+        if(instancesResponse.status != 200){
```

Comment:
Use double equals in JS.

https://stackoverflow.com/questions/5112934/javascript-triple-equals

Reply:
@adinhodovic you could just run `yarn lint:fix` to format the code. It will fix the quotes issues